### PR TITLE
RSWEB-8951 feat(newpattern): addition of service level teaser

### PR DESCRIPTION
**Pull request by runrog**
Addition of Service level teaser pattern.
Seen here: 
http://localhost:3000/derek/solutions/solutions

### CODE REVIEW

**Comment by WidgetsBurritos** [comment]
[run ci]

File: styleguide/derek/pattern-components/service-level-teaser/_service-level-teaser.scss [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  width: 100%;
+
+  &:hover {
+    background-color: $gray-lightest;
+    text-decoration: none;
+  }
+
+  a {
+    @include standard-link;
+  }
```
Comment (WidgetsBurritos) [inline styleguide/derek/pattern-components/service-level-teaser/_service-level-teaser.scss]: Insert typical comment about, is this nested anchor absolutely necessary here? Shouldn't we just be targetting `.serviceLevelTeaser-bottomLink`, or am I misunderstanding the use case?

File: styleguide/derek/pattern-components/service-level-teaser/service-level-teaser.ejs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  </div>
+</div>
+
+<div class="half-padding container">
+  <h3>Service Level Teasers 2 only (default)</h3>
+</div>
+
+<!-- Teaser with only 2 option -->
+<div class="half-padding">
+  <div class="container">
```
Comment (WidgetsBurritos) [inline styleguide/derek/pattern-components/service-level-teaser/service-level-teaser.ejs]: It's worth noting that either this PR, or #444 will need to be adjusted (depending on merge order), to allow for usage of the new `.derek-paddedContainer` class. It doesn't require immediate action, but we'll need to make sure we remember that come merge time.

File: styleguide/derek/pattern-components/service-level-teaser/_service-level-teaser.scss (hunk 1 above)
Comment (runrog) [inline styleguide/derek/pattern-components/service-level-teaser/_service-level-teaser.scss]: @WidgetsBurritos oh nice catch. This is not needed.

File: styleguide/derek/pattern-components/service-level-teaser/service-level-teaser.ejs (hunk 2 above)
Comment (runrog) [inline styleguide/derek/pattern-components/service-level-teaser/service-level-teaser.ejs]: Ok noted.

**Pull request merged**